src/probe/probe.c

```diff
@@ -926,6 +958,14 @@ static void probe_cb_free(void *arg, enum notify_id type, void *data)
 		tr_err(&pr_tr, "probe_cb_free(): probe_point_remove() failed");
 }
 
+static bool probe_purpose_needs_ext_dma(uint32_t purpose)
+{
+	if (purpose == PROBE_PURPOSE_EXTRACTION || purpose == PROBE_PURPOSE_LOGGING)
+		return true;
+
+	return false;
```

Comment:
nitpick: `return purpose == PROBE_PURPOSE_EXTRACTION || purpose == PROBE_PURPOSE_LOGGING;`

Reply:
Change in V6.